chapter_convolutional-neural-networks/conv-layer_vn.md

```diff
@@ -287,8 +287,8 @@ Designing an edge detector by finite differences `[1, -1]` is neat if we know th
 However, as we look at larger kernels, and consider successive layers of convolutions, it might be impossible to specify precisely what each filter should be doing manually.
 -->
 
-Thiết kế một máy dò cạnh bằng hiệu hữu hạn giữa `[1, -1]` thì rất tuyệt nếu ta biết điều này chính xác là những gì ta đang tìm kiếm.
-Tuy nhiên, khi xét tới các bộ lọc lớn hơn và các tầng tích chập liên tiếp, sẽ không khả thi để chỉ định thủ công chính xác mỗi bộ lọc nên làm gì.
+Việc thiết kế bộ phát hiện biên bằng sai phân `[1, -1]` thì khá là gọn gàng nếu ta biết chính xác đây là những gì cần làm.
```

Comment:
từ `sai phaan` này khá là mới với mình. Bạn kiếm được nó ở đâu vậy?

Reply:
mình thấy định nghĩa toán học của [sai phân](https://vi.wiktionary.org/wiki/sai_ph%C3%A2n#Ti%E1%BA%BFng_Vi%E1%BB%87t) và [difference](https://en.wikipedia.org/wiki/Finite_difference#Types) giống nhau. Có lẽ vẫn cần thêm từ `hữu hạn`.
```suggestion
Việc thiết kế bộ phát hiện biên bằng sai phân hữu hạn `[1, -1]` thì khá gọn gàng nếu ta biết chính xác đây là những gì cần làm.
```